net/strongswan/Makefile

```diff
@@ -502,9 +504,20 @@ endef
 define Package/strongswan-ipsec/install
 	$(INSTALL_DIR) $(1)/etc/ $(1)/usr/sbin
 	$(INSTALL_CONF) $(PKG_INSTALL_DIR)/etc/ipsec.conf $(1)/etc/
+	echo -e "\ninclude /var/ipsec/ipsec.conf" >> $(1)/etc/ipsec.conf
 	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/sbin/ipsec $(1)/usr/sbin/
 endef
 
+define Package/strongswan-ipsec/postinst
```

Comment:
This is not mentioned at all in the commit message. It's also completely unrelated to the changes in this commit, so this must go in a different commit, with the commit message explaining what/why.

Reply:
I'm with @stintel on that one.

Reply:
Okay, hopefully @stintel and @luizluca are okay with the added description in the new commit message.

Reply:
Looks okay. It's legacy anyway with that.